src/core/edgeNetwork/injectSendEdgeNetworkRequest.js

```diff
@@ -51,7 +52,11 @@ export default ({
         const endpointDomain = request.getUseIdThirdPartyDomain()
           ? ID_THIRD_PARTY_DOMAIN
           : edgeDomain;
-        const url = `https://${endpointDomain}/${edgeBasePath}/${apiVersion}/${request.getAction()}?configId=${edgeConfigId}&requestId=${request.getId()}`;
+        const locationHint = getLocationHint();
+        const edgeBasePathWithLocationHint = locationHint
+          ? `${edgeBasePath}/${locationHint}`
+          : edgeBasePath;
+        const url = `https://${endpointDomain}/${edgeBasePathWithLocationHint}/${apiVersion}/${request.getAction()}?configId=${edgeConfigId}&requestId=${request.getId()}`;
```

Comment:
Should re rename configId to edgeConfigId at this point? Or does that require too many changes?

Reply:
Are you saying that Konductor accepts edgeConfigId query string parameter now? I was not aware of this.